fake_service/schema/tasks.py

```diff
@@ -1,17 +1,25 @@
 from celery import shared_task
 from django.core.cache import cache
-
 from schema.services import FakeDataWrite
+from schema.utils import generate_cache_task_key
 
 
 @shared_task()
-def download_dataset_task(cache_task_key: str, schema_id: str, count: str, user_id: str):
+def download_dataset_task(cache_task_key: str, schema_id: str, row_nums: str, user_id: str):
     try:
         cache.set(cache_task_key, {"is_importing": False, "errors": []}, timeout=300)
-        FakeDataWrite().write_data_to_db_and_remove_from_disk(schema_id=schema_id, count=count, user_id=user_id)
+        FakeDataWrite().write_data_to_db_and_remove_from_disk(schema_id=schema_id, row_nums=row_nums, user_id=user_id)
     except Exception as exc:
         cache.set(
-            cache_task_key, {"is_importing": False, "errors": [exc.args]}, timeout=120
+            cache_task_key, {"in_process": False, "errors": [exc.args]}, timeout=120
```

Comment:
still bad naming. I think would be better to name this key "generating" or may be "downloading" or even "in_progress". Or may be change it to "status" and isnted of bool values use constants like "in_progress", "done" etc